site/installation/migration/01-migration.md

```diff
@@ -0,0 +1,278 @@
+## Migration from kpt `v0.39.0` to kpt `v1.0`
```

Comment:
if you're keeping in /site not sure why it can't be in `installation/migration.md`